infrastructure/mongo-init.js

```diff
@@ -3,7 +3,8 @@ db.createUser({
   pwd: 'password',
   roles: [
     { role: 'readWrite', db: '__inxt-network' },
-    { role: 'readAnyDatabase', db: 'admin' }
+    { role: 'readAnyDatabase', db: 'admin' },
```

Comment:
Why do you need this?

Reply:
Without that role the `database-viewer` container fails with
```
MongoServerError: not authorized on config to execute command { listCollections: 1, filter: {}, cursor: {}, nameOnly: false, authorizedCollections: false, lsid: { id: UUID("61969390-3549-4bd0-835d-61075db37558") }, $db: "config" }
```

I added the `clusterMonitor` role to give readonly access to the database
https://www.mongodb.com/docs/v4.4/reference/built-in-roles/#mongodb-authrole-clusterMonitor